### PR TITLE
Adding blueprint

### PR DESCRIPTION
**Pull request by jacob-shaw-genesys**
Adding the complete partial transcription blueprint

### CODE REVIEW

File: blueprint/index.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+title: Develop an Integration App That Utilizes Partial Transcript Notifications
+author: jacob.shaw
+indextype: blueprint
+icon: blueprint
+image: images/5.png
```
Comment (timsmithgenesys) [inline blueprint/index.md]: There's no image by this name included in the PR

File: blueprint/index.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+* [Configuring the React Project to use Genesys Cloud SDK](#configuring-the-react-project-to-use-genesys-cloud-sdk "How to integrate the Genesys Cloud SDK")
+* [Additional resources](#additional-resources "Goes to the Additional resources section")
+
+
+![Partial Transcript App Flowchart](images/flowchart.png)
+
+## Solution Components
+
+- **Genesys Cloud** - A suite of Genesys cloud services for enterprise-grade communications, collaboration, and contact center management. You deploy the Chat Translator solution in Genesys Cloud.
+- **Genesys AppFoundry** - The Genesys app marketplace for solutions that run on the Genesys Cloud platform. You download the integration used in this solution from the Genesys AppFoundry.
```
Comment (timsmithgenesys) [inline blueprint/index.md:27]: Add a link to the integration here

File: blueprint/index.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+- **Client Application integration** - The Genesys Cloud integration that embeds third-party webapps via iframe in the Genesys Cloud UI.
+
+### Software Development Kit (SDK)
+
+- **Platform API Javascript Client** - The sample app employs React+Typescript, thus the javascript SDK is used here. However, the same functionality could be achieved using other languages. Find the [Platform API Javascript Client](https://github.com/MyPureCloud/platform-client-sdk-javascript) here: https://github.com/MyPureCloud/platform-client-sdk-javascript
```
Comment (timsmithgenesys) [inline blueprint/index.md]: The link and URL is redundant

Comment (timsmithgenesys) [inline blueprint/index.md]: TypeScript should always be spelled as one word with a capital T and S. There are 4 instances of the word in this document

File: blueprint/index.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+const analyticsApi = new platformClient.AnalyticsApi();
+const tokensApi = new platformClient.TokensApi();
+const routingApi = new platformClient.RoutingApi();
+const presenceApi = new platformClient.PresenceApi();
+const conversationsApi = new platformClient.ConversationsApi();
+```
+
+## Additional Resources
+
+* [Genesys Cloud Platform SDK - Javascript](https://developer.genesys.cloud/api/rest/client-libraries/javascript/)
```
Comment (timsmithgenesys) [inline blueprint/index.md]: Drop the domain from links to the dev center, just use the path

File: blueprint/index.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+4. Click the Configuration tab.
+5. In the Application URL box, type the URL of the web application. Be sure to specify the full URL.
+`https://localhost:3000`
+6. In the Application Type dropdown, select **widget**
+7. To limit access to specific groups of agents, in Group Filtering, select the groups that can use the widget.
+![Client Application Integration Config](images/integration-config.png)
+8. Activate the client application
+
+### Test the Solution
+1. Select a queue that you are a member of. Note that setting up a test queue with only yourself as a member is a quick solution here, since this guarantees that inbound calls to the queue will be assigned to you.  See [Create and configure queues](https://help.mypurecloud.com/articles/create-queues/) in the Genesys Cloud Resource Center for more.
```
Comment (timsmithgenesys) [inline blueprint/index.md]: This should say "Select a queue that you are a member of _and are activated in_." to remind people to check their activation status in the queue if they aren't getting things assigned.

File: blueprint/index.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+8. Answer the incoming call in the Genesys Cloud app.
+9. Open the Apps in the side navigation bar.
+10. Open the client application which was set up in previous steps.
+11. Find your queue in the **Active Conversation Dashboard** and expand to find your active conversation's listing.
+
+## Sample App Overview
+
+### Genesys Cloud Utils
+
+The `genesysCloudUtils` file contains the intermediate functions that in turn call Genesys Cloud SDK methods. These functions return promises that are handled upon resolution in the file and in the invoking components themselves.
```
Comment (timsmithgenesys) [inline blueprint/index.md]: include the file extension and relative path in the repo when referencing a specific file (`genesysCloudUtils`)

File: public/index.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (timsmithgenesys) [inline public/index.html]: Set a title for the app

File: public/manifest.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (timsmithgenesys) [inline public/manifest.json]: set short_name and name and remove the icon references you aren't using in the project

File: src/App.css [hunk 9]
```diff
@@ -0,0 +1,7 @@
+body {
+  text-align: center;
+  font-family: "Roboto",sans-serif;
```
Comment (timsmithgenesys) [inline src/App.css:3]: Are you importing the roboto font from somewhere? I don't see the font import. If you haven't, you can generate a link with the styles you need here: https://fonts.google.com/?query=roboto or grab the link from the index.html file for the dev center.

File: src/components/accordion/Accordion.tsx [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+import './Accordion.scss';
+
+interface IAccordionProps {
+    sections: AccordionData[],
+    innerClass?: string,
+    outerClass?: string
+}
+
+export interface AccordionData {
+	heading: React.ReactNode;
```
Comment (timsmithgenesys) [inline src/components/accordion/Accordion.tsx]: indentation

File: src/components/accordion/Accordion.tsx [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+    contentClass?: string
+}
+
+/**
+ * Wrapper component for a collection of accordion sections
+ * 
+ * @param props 
+ * @returns 
+ */
+export function Accordion(props: IAccordionProps) {
```
Comment (timsmithgenesys) [inline src/components/accordion/Accordion.tsx:25]: You can replace this component with the genesys-react-components library if you like. Not required. https://www.npmjs.com/package/genesys-react-components

File: src/logo.svg [hunk 12]
```diff
@@ -0,0 +1 @@
...21.2 430c-9.3-9.6-18.6-20.3-27.8-32 9 .4 18.2.7 27.5.7 9.4 0 18.7-.2 27.8-.7-9 11.7-18.3 22.4-27.5 32zm-74.4-58.9c-14.2-2.1-27.9-4.7-41-7.9 3.7-12.9 8.3-26.2 13.5-39.5 4.1 8 8.4 16 13.1 24 4.7 8 9.5 15.8 14.4 23.4zM420.7 163c9.3 9.6 18.6 20.3 27.8 32-9-.4-18.2-.7-27.5-.7-9.4 0-18.7.2-27.8.7 9-11.7 18.3-22.4 27.5-32zm-74 58.9c-4.9 7.7-9.8 15.6-14.4 23.7-4.6 8-8.9 16-13 24-5.4-13.4-10-26.8-13.8-39.8 13.1-3.1 26.9-5.8 41.2-7.9zm-90.5 125.2c-35.4-15.1-58.3-34.9-58.3-50.6 0-15.7 22.9-35.6 58.3-50.6 8.6-3.7 18-7 27.7-10.1 5.7 19.6 13.2 40 22.5 60.9-9.2 20.8-16.6 41.1-22.2 60.6-9.9-3.1-19.3-6.5-28-10.2zM310 490c-13.6-7.8-19.5-37.5-14.9-75.7 1.1-9.4 2.9-19.3 5.1-29.4 19.6 4.8 41 8.5 63.5 10.9 13.5 18.5 27.5 35.3 41.6 50-32.6 30.3-63.2 46.9-84 46.9-4.5-.1-8.3-1-11.3-2.7zm237.2-76.2c4.7 38.2-1.1 67.9-14.6 75.8-3 1.8-6.9 2.6-11.5 2.6-20.7 0-51.4-16.5-84-46.6 14-14.7 28-31.4 41.3-49.9 22.6-2.4 44-6.1 63.6-11 2.3 10.1 4.1 19.8 5.2 29.1zm38.5-66.7c-8.6 3.7-18 7-27.7 10.1-5.7-19.6-13.2-40-22.5-60.9 9.2-20.8 16.6-41.1 22.2-60.6 9.9 3.1 19.3 6.5 28.1 10.2 35.4 15.1 58.3 34.9 58.3 50.6-.1 15.7-23 35.6-58.4 50.6zM320.8 78.4z"/><circle cx="420.9" cy="296.5" r="45.7"/><path d="M520.5 78.1z"/></g></svg>
```
Comment (timsmithgenesys) [inline src/logo.svg]: this file can be removed (default react "logo")

File: src/utils/genesysCloudUtils.ts [hunk 13]
```diff
@@ -0,0 +1,228 @@
...
+    onQueueUsers: number
+}
+
+const searchApi = new platformClient.SearchApi();
+const usersApi = new platformClient.UsersApi();
+const analyticsApi = new platformClient.AnalyticsApi();
+const tokensApi = new platformClient.TokensApi();
+const routingApi = new platformClient.RoutingApi();
+const presenceApi = new platformClient.PresenceApi();
+// const conversationsApi = new platformClient.ConversationsApi();
```
Comment (timsmithgenesys) [inline src/utils/genesysCloudUtils.ts]: remove commented out code

File: src/utils/genesysCloudUtils.ts [hunk 14]
```diff
@@ -0,0 +1,228 @@
...
+ * In constrast, Org presences are not hardcoded.
+*/
+const offlinePresenceId = 'ccf3c10a-aa2c-4845-8e8d-f59fa48c58e5';
+
+const client = platformClient.ApiClient.instance;
+const { clientId, redirectUri } = clientConfig;
+
+const cache: any = {};
+
+export function authenticate() {
```
Comment (timsmithgenesys) [inline src/utils/genesysCloudUtils.ts:25]: This file, specifically,  should be heavily commented. State the purpose of every function and explain every Genesys Cloud API request (briefly). This is the part of the app that people are going to reference to learn how to use our APIs, so being thorough with commenting will help guide people through it.

**Review by timsmithgenesys — CHANGES_REQUESTED** [review]

**Comment by timsmithgenesys** [comment]
Consider publishing this page using github pages so there's a live demo of it available. Publishing the app is super easy using the [gh-pages](https://www.npmjs.com/package/gh-pages) package. Let me know if you need me to turn github pages on for the repo or have any questions; not sure if you have permissions to do that.

**Review by timsmithgenesys — APPROVED** [review]

**Comment by timsmithgenesys** [comment]
approve

**Review by carnellj — APPROVED** [review]

**Review by carnellj — APPROVED** [review]

**Pull request merged**